tabs/osd.html

```diff
@@ -102,6 +102,7 @@ <h1 class="tab_title">Font presets:</h1>
                         <button data-font-file="large">Large</button>
                         <button data-font-file="extra_large">Extra Large</button>
                         <button data-font-file="betaflight">Betaflight</button>
+                        <button data-font-file="mugzless">Betaflight</button>
```

Comment:
This puts 2 'Betaflight' entries into the list. Better change the name to something else.